### DISCUSSION TITLE
SKU:MRB2801 - Is this screen compatible

### DISCUSSION THREAD

**Discussion by OptimusGREEN** [Bodmer/TFT_eSPI#1142] [Q&A - Touch]
Hi,

Sorry, new to touch screens etc but looking to get [this screen](http://www.lcdwiki.com/2.8inch_16BIT_Module_ILI9341_SKU:MRB2801) working with an esp32-wroom-32 module.

I've looked high and low for a guide because my logic and common sense configs have only ended with a white screen

Is this 1 million pin display compatible?

**Comment by Bodmer** [comment]
Yes  the screen is an ILI9341 type so should work fine in 8 bit parallel mode. You will need another touch library though to use the touch screen.

**Reply by OptimusGREEN** [reply]
Hi thanks for the reply.

What would the wiring be for this? The reason I ask is I only got a white screen, I can worry about the touch later.
